docs/theme.config.tsx

```diff
@@ -188,7 +190,12 @@ const config: DocsThemeConfig = {
       </>
     ),
   },
+  components: {
+    pre: (props) => <pre {...props} data-pagefind-weight=".5" />,
```

Comment:
Downranking code snippets. Motivation is that our previous results were cluttered with meaningless results where the context of the snippet wasn't addressing the search query.

e.g. Search for `dependsOn`, get results for "How to Test" because the snippet has a `dependsOn`.